backend/src/main/java/com/woowacourse/kkogkkog/application/MemberService.java

```diff
@@ -36,7 +36,7 @@ public Long save(MemberCreateRequest memberCreateRequest) {
 
     public MembersResponse findAll() {
         List<MemberResponse> memberResponses = memberRepository.findAll().stream()
-            .map(it -> new MemberResponse(it.getId(), it.getEmail(), it.getNickname()))
+            .map(it -> MemberResponse.of(it))
             .collect(toList());
```

Comment:
메서드 참조를 사용하는건 어떨까요? 

`MemberResponse .of(it)`이라고 하는게 가독성은 더 있지만, 
인텔리제이에서 메서드 참조를 사용하라고 계속 표시될 것 같고, 바이트코드상으로도 더 이점이 있는 것으로 알아요.

```suggestion
            .map(MemberResponse::of)
            .collect(toList());
```